src/Application/FileProcessor/PhpFileProcessor.php

```diff
@@ -80,12 +84,19 @@ public function process(File $file, Configuration $configuration): array
             // important to detect if file has changed
             $this->printFile($file, $configuration);
 
+            $fileHasChanged = $fileHasChanged || $file->hasChanged();
+
             if ($file->hasChanged()) {
```

Comment:
Repetitive `$file->hasChanged()` with previous line 87 can be moved to variable.